src/ralph/back_office/migrations/0010_auto_20181123_1037.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
are you sure this migration is still relevant? Shouldn't it be deleted? You don't have any changes in backoffice app